components/PageNavigation.vue

```diff
@@ -295,6 +296,16 @@
             color: $white;
           }
         }
+        span {
+          overflow: hidden;
+          white-space: nowrap;
+          display: block;
+          text-overflow: ellipsis;
+          height: 1.5rem;
```

Comment:
Remove this height as it's unnecessary when display is set to flex as mentioned above.